uniffi_bindgen/src/bindings/kotlin/gen_kotlin/object.rs

```diff
@@ -0,0 +1,126 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is another good example of the pattern.

We have a `CodeType` to allow things to move across the FFI, a `CodeDeclaration` to declare the foreign language version of the object, and a `CodeDeclaration` that declares the runtime for objects.